### ISSUE TITLE
Clarify what should be selected if a SQL Server license was purchased with a CSP Software Subscription?

### ISSUE THREAD

**Issue by mtwelve** [MicrosoftDocs/SupportArticles-docs#523]
No feedback button on the page so here is a link - https://docs.microsoft.com/en-us/troubleshoot/sql/linux/choosing-licensing-model-using-mssql-conf

Can you clarify what should be selected if a SQL Server license was purchased with a CSP Software Subscription?

The wording of this page is currently:

_If you obtained your license through Volume Licensing as part of an Enterprise Agreement or through your MSDN subscription, you must select from among options 4 through 7. If you purchased a Standard edition through a retail channel, you must select option 8._

CSP Software Subscriptions are neither part of an Enterprise Agreement or an MSDN Subscription however when going to install we are directed to use mssql-conf Setup tool

**Comment by HaiyingYu** [comment]
@mtwelve Thanks for the comment. I will check with our engineers, and get back to you.

**Comment by mtwelve** [comment]
Any updates available on this one? Keen to reassure our customer which option they should pick when installing to Linux after purchasing a CSP Software Subscription.

**Comment by HaiyingYu** [comment]
> Any updates available on this one? Keen to reassure our customer which option they should pick when installing to Linux after purchasing a CSP Software Subscription.

Not yet. Sorry about that. I will check the status and get back to you.

**Comment by tejasaks** [comment]
It's probably a good idea to validate this with the CSP provider themselves. In most cases it would be same as EA or MSDN subscription and you should use option 4 to 7. But it's possible that the CSP provider is also a retailer/reseller of Microsoft software and this particular solution was offered under that ambit (and thus enabled by a 25-character key). It would be presumptuous for us to comment on that without that additional information, which the CSP provider can confirm quite easily.

**Comment by mtwelve** [comment]
I am from the MSP and I am trying to get confirmation on behalf of our customer.

I have raised this through Partner Centre support who directed me to "Technical Presales and Deployment team". They in turn offered no resolution on a ticket open for over 3 months and with many prompts, chases and offers to support in any way they required. There is no CSP/Partner Centre specific documentation that answers this question.

I have turned to the documentation, as the page above is where you are directed when retrieving the required details for a customer via Partner Centre for a Linux SQL Server installation licensed via a CSP Software Subscription.

I'd assume it is option 4 to 7 however, like yourself, I don't want to be making assumptions.
